Makefile

```diff
@@ -335,3 +335,7 @@ gen-docs:
 .PHONY: test-e2e
 test-e2e: local
 	$(MAKE) -C test/e2e run
+
+.PHONY: test-e2e-testify
+test-e2e-testify: local
```

Comment:
I don't think that we should have a separate target for running these tests. Can we combine it all into the `test-e2e` target? Otherwise, I think it would be really easy to miss running one of them.

Reply:
Answer for future reference because I'm closing this PR. 

I'm afraid when I put the testify and ginkgo test files together, it caused problems and the tests were not able to run using `go test`.